### PR TITLE
ui: translations

### PR DESCRIPTION
**Pull request by epoberezkin**

### CODE REVIEW

**Comment by summoner001** [comment]
Sorry for my mistake in Dutch translation.

In this pr, that one word will be corrected. I accidentally thought I was in the Hungarian translation on weblate.

Sorry.

File: apps/ios/ru.lproj/Localizable.strings [hunk 1]
```diff
@@ -936,7 +936,7 @@
 "Clear verification" = "Сбросить подтверждение";
 
 /* No comment provided by engineer. */
-"Color chats with the new themes." = "Добавьте цвета к чатам в настройках.";
+"Color chats with the new themes." = "Добавьте цвета к чатам в настройках тем.";
```
Comment (epoberezkin) [inline apps/ios/ru.lproj/Localizable.strings:939]:
```suggestion
"Color chats with the new themes." = "Добавьте цвета к чатам в настройках.";
```

File: apps/ios/ru.lproj/Localizable.strings [hunk 2]
```diff
@@ -1459,7 +1459,7 @@
 "Delete queue" = "Удаление очереди";
 
 /* No comment provided by engineer. */
-"Delete up to 20 messages at once." = "Удаляйте до 20 сообщений за раз.";
+"Delete up to 20 messages at once." = "Удаляйте до 20 сообщений одновременно.";
```
Comment (epoberezkin) [inline apps/ios/ru.lproj/Localizable.strings:1462]:
```suggestion
"Delete up to 20 messages at once." = "Удаляйте до 20 сообщений за раз.";
```

File: apps/ios/ru.lproj/Localizable.strings [hunk 3]
```diff
@@ -3358,7 +3358,7 @@
 "Private message routing" = "Конфиденциальная доставка сообщений";
 
 /* No comment provided by engineer. */
-"Private message routing 🚀" = "Конфиденциальная доставка 🚀";
+"Private message routing 🚀" = "Конфиденциальная доставка сообщений 🚀";
```
Comment (epoberezkin) [inline apps/ios/ru.lproj/Localizable.strings:3361]:
```suggestion
"Private message routing 🚀" = "Конфиденциальная доставка 🚀";
```

**Pull request merged**